### DISCUSSION TITLE
Make non-music mode default?

### DISCUSSION THREAD

**Discussion by ballo** [Thunderarea/music-mode-for-youtube#27] [General]
I'm constantly having to switch this extension on and off because if it's on, the default is music mode. If the default was not music mode I could select which tabs I want to use it in. 

A button on the modified web page to switch modes would also be nice.

**Comment by Thunderarea** [comment]
I did not understand what exactly is the problem. What do you mean by "_if it's on, the default is music mode_"? You can have the extension turned off from the extension's menu or from the extension's options and specifically the option:

> **Enabled by default**
> _When you open a new tab the extension will be enabled by default_

Therefore, whenever you want to enable the extension for a tab you can do it either from the extension's menu or by using the shortcut "Ctrl + M" (for Chromium browsers) or "Ctrl + ↑" for Firefox. That's how I use the extension as well. If you have any further questions don't hesitate to contact me again.

**Comment by ballo** [comment]
I have no idea where you are quoting the "Enabled by default" option, but the command-M shortcut works on a Mac and works well. Thanks.

The problem I was having is I could choose to have the extension enabled or disabled, but if I turned it on I could only selectively disable it for the tabs where I didn't want it enabled and not visa versa (default disabled). This is still the case when using the settings:

<img width="301" alt="Screenshot 2024-01-21 at 12 11 18 PM" src="https://github.com/Thunderarea/music-mode-for-youtube/assets/631311/b2375466-093c-4304-91ec-addd5f27d7d4">

The command-M shortcut makes this easier, however, so for at least my use case it works fine. Without the shortcut it's a nightmare.

**Comment by Thunderarea** [comment]
First of all, you can choose to have the extension disabled by the menu:
![image](https://github.com/Thunderarea/music-mode-for-youtube/assets/27777837/3e838c8e-5b96-40ee-8705-18805a671331)
or by the options (menu → Options):
![image](https://github.com/Thunderarea/music-mode-for-youtube/assets/27777837/3cf9c97a-dd9c-4cae-bada-20245c911ecd)
They are the exact same option. 
Then when you  go to a tab that you want the extension to be enabled you can either use the menu to enable it only for the current tab:
![image](https://github.com/Thunderarea/music-mode-for-youtube/assets/27777837/f1bebc6b-afba-4221-bd04-be0cc46344c2)
or use the shortcut Ctrl + M that does the exact same thing in an easier way (you don't have to open the menu etc etc ...)
The shortcut Ctrl + M does not enable or disable the extension by default. It just toggles its function. So, if it is enabled it will disable it and if it is disabled it will enable it (but only for the current tab). 
The fact that you have the extension disabled it does not mean that you cannot enable it for a tab or page. It just means that it will not be enabled by default when you open a new tab. 
I hope that clarifies everything. Don't hesitate to reach out if you have more questions!
